packages/core/src/base/typography/link.scss

```diff
@@ -8,7 +8,7 @@
   // would cause the selector's specificity to be higher than most other component
   // selectors (i.e. ds-c-button), necessitating overly specific selectors anytime
   // a developer wanted to change an anchor's color property.
-  a {
+  a, .ds-c-link {
```

Comment:
This is inside the `$ds-include-base-html-rulesets` statement, when we actually want `.ds-c-link` to be always defined. One way we could potentially organize this is by using `extend` and sass placeholders

```
%link {
   color: $color-primary;
   &:hover { ... }
   ...
} 

%link-inverse {...}

.ds-c-link {
   @extend %link;
}

.ds-c-link--inverse {
   @extend %link-inverse;
}

@if $ds-include-base-html-rulesets {
   a {
      @extend %link;
   }

   .ds-base--inverse a {
       @extend %link-inverse;
   }
}